test/transcoding/kernel_arg_type_qual.ll

```diff
@@ -0,0 +1,43 @@
+; RUN: llvm-as %s -o %t.bc
+; RUN: llvm-spirv -preserve-ocl-kernel-arg-type-metadata-through-string %t.bc -o - -spirv-text | FileCheck %s --check-prefix=CHECK-SPIRV
+; RUN: llvm-spirv %t.bc -o - -spirv-text | FileCheck %s --check-prefix=CHECK-SPIRV-NEGATIVE
+; RUN: llvm-spirv -preserve-ocl-kernel-arg-type-metadata-through-string %t.bc -o %t.spv
+; RUN: llvm-spirv -r %t.spv -o %t.rev.bc
+; RUN: llvm-dis < %t.rev.bc | FileCheck %s --check-prefix=CHECK-LLVM
```

Comment:
The CHECK-LLVM prefix here should be renamed to make it clearer that it is testing nonstandard functionality.

Reply:
Looks okay.  In hindsight, it may have been better to place all of these tests for nonstandard metadata into their own subdirectory under transcoding, perhaps with a README or at least name that indicates clearly that this metadata is nonstandard.